frontend/cypress/tests/chartView/chartView_toolBar_searchBox.cy.js

```diff
@@ -33,4 +33,131 @@ describe('search bar', () => {
       expect(children).to.equal(1);
     });
   });
+
+  it('searching by non-existent tag shows no results', () => {
+    cy.get('#app #tab-resize .tab-close').click();
+    cy.get('#summary-wrapper input[type=text]')
+      .type('tag: asdfghjkl')
+      .type('{enter}');
+
+    // Enter does not work. Related issue: https://github.com/cypress-io/cypress/issues/3405
+    // Let's manually submit form
+    cy.get('#summary-wrapper form.summary-picker')
+      .submit();
+
+    cy.get('#summary-wrapper #summary-charts')
+      .should('be.empty');
+  });
+
+  it("searching tag that only exists in one author's commits shows one result", () => {
+    cy.get('#app #tab-resize .tab-close').click();
+    cy.get('#summary-wrapper input[type=text]')
+      .type('tag: v1.8')
+      .type('{enter}');
+
+    // Enter does not work. Related issue: https://github.com/cypress-io/cypress/issues/3405
+    // Let's manually submit form
+    cy.get('#summary-wrapper form.summary-picker')
+      .submit();
+
+    cy.get('.summary-chart__title--name')
+      .should('have.length', 1)
+      .and('contain', 'Eugene (eugenepeh)');
+
+    cy.get('.icon-button.fa-list-ul')
+      .should('exist')
+      .first()
+      .click();
+
+    cy.get('.zoom__title--tags > .tag span')
+      .should('contain', 'v1.8');
+  });
+
+  it("searching tag that only exists in two authors' commits shows two results", () => {
+    cy.get('#app #tab-resize .tab-close').click();
+    cy.get('#summary-wrapper input[type=text]')
+      .type('tag: v1.10')
+      .type('{enter}');
+
+    // Enter does not work. Related issue: https://github.com/cypress-io/cypress/issues/3405
+    // Let's manually submit form
+    cy.get('#summary-wrapper form.summary-picker')
+      .submit();
+
+    cy.get('.summary-chart__title--name')
+      .should('have.length', 2)
+      .and('contain', 'Eugene (eugenepeh)')
+      .and('contain', 'James (jamessspanggg)');
+
+    cy.get('.icon-button.fa-list-ul')
+      .should('exist')
+      .each(($ele) => {
+        cy.wrap($ele).click();
+        cy.get('.zoom__title--tags > .tag span')
+          .should('contain', 'v1.10');
+      });
+  });
+
+  it("search field doesn't start with 'tag:' prefix but still contains it shows no results", () => {
+    cy.get('#app #tab-resize .tab-close').click();
+    cy.get('#summary-wrapper input[type=text]')
+      .type('v1.10 tag: v1.10')
+      .type('{enter}');
+
+    // Enter does not work. Related issue:
```

Comment:
The related GitHub issue is missing for some of the comments in this file.

Reply:
the comment was copied from the existing tests in the file. if that's the case, maybe i can remove the comment since submitting seems self-explanatory anyway?

Reply:
Sure you can remove them